app/helpers/arclight_helper.rb

```diff
@@ -167,11 +167,12 @@ def fields_have_content?(document, field_accessor)
   #
   # @return [Repository]
   def repository_faceted_on
-    return unless try(:search_state)
+    search_fields = params.fetch('f', nil)
```

Comment:
I think Blacklight's `facet_field_in_params?` helper might do the trick here?

And, retrieving the value might just be `Array(facet_params['repository_sim']).first`?

Reply:
I have the impression that checking that there is only one repository is deliberate, but I don't actually know if we need to handle the theoretical case of having two repositories in the facets, since as far as I know they are 'and' rather than 'or' when you filter by them.